.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ orbs:
 executors:
  machine_executor_amd64:
    machine:
-     image: ubuntu-2204:2022.04.2
+     image: default
```

Comment:
Was there an issue with locking to a specific image?

Reply:
Yup, circleCI deprecated this specific version, and the suggested version by them is default.
[Post about this deprecation.](https://discuss.circleci.com/t/linux-image-deprecations-and-eol-for-2024/50177)